xml/System.Net.Security/CipherSuitesPolicy.xml

```diff
@@ -39,9 +39,20 @@
         <Parameter Name="allowedCipherSuites" Type="System.Collections.Generic.IEnumerable&lt;System.Net.Security.TlsCipherSuite&gt;" />
       </Parameters>
       <Docs>
-        <param name="allowedCipherSuites">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="allowedCipherSuites">The collection of cipher suites allowed in this policy for negotiation.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Net.Security.CipherSuitesPolicy" /> class that can be used on a client or server authentication.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+> [!NOTE]
+> Defining a cipher suite policy on <xref:System.Net.Security.SslStream> authentication will prevent the OS of the ability to decide which are the best cipher suites to negotiate with and will need of your attention to manually check and update this code. It is strongly recommended to avoid the use of this property and rely on your constantly updated OS policy.
+
+## Remarks
+
+Unlike the OS underlying SSL/TLS implementation, the order of the <xref:System.Net.Security.TlsCipherSuite> elements in `allowedCipherSuites` doesn't guarantee their priority in the client-server negotiation. You can only initialize an instance of this class if you're using Linux systems with OpenSSL 1.1.1 or higher or macOS .
+
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.PlatformNotSupportedException">Platform is not a Linux system with OpenSSL 1.1.1 or higher or a macOS.</exception>
```

Comment:
Is this the right wording? If the exception can be thrown on macOS for have an earlier version than OpenSSL 1.1.1,f it should read, "Platform is not a Linux system or macOS with OpenSSL 1.1.1 or higher." The same question applies to the Remarks section.

Reply:
We actually try to say (Linux with Open SSL 1.1.1) or (macOS). 
`Open SSL v1.1.1` is only required on linux; please see https://github.com/dotnet/dotnet-api-docs/pull/2822#discussion_r307029105.

Reply:
It is the right wording, then. Thanks for clarifying, @Jozkee.